Makefile

```diff
@@ -34,3 +34,7 @@ clean:
 integration:
 	make clean && make
 	python reverse_proxy_integration.py
+
+integration_302:
+	make clean && make
+	python reverse_proxy_302_integration.py
```

Comment:
Hi, the reason we put it under two different build targets is because the submission link for assignment 7 required us to provide the command to run test a regular proxy and a 302 proxy, so we thought it would be more convenient for the TA if we separated the two.